tests/ast/data/DataframeNaFunctions.test

```diff
@@ -38,22 +38,26 @@ res3 = res1.na.drop("all")
 
 res4 = res1.na.drop("any", 42)
 
-res5 = res1.na.drop("any", ["STR"])
+res5 = res1.limit(0, 0)
```

Comment:
This is wrong. It's possible you might need to enable AST suppression in `limit()`.

Reply:
done